lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__grid_mappings.py

```diff
@@ -0,0 +1,884 @@
+        #     003 : fc_provides_coordinate_(latitude)
+        #     004 : fc_provides_coordinate_(longitude)
+        #     005 : fc_build_coordinate_(latitude)
+        #     006 : fc_build_coordinate_(longitude)
+        # Notes:
+        #     grid-mapping: regular latlon
+        #     dim-coords: lat+lon
+        #     coords-build: standard latlon coords (with latlon coord-system)
+        result = self.run_testcase()
+        self.check_result(result)
+
+    def test_missing_latlon_radius(self):
+        # Lat-long with a missing earth-radius causes an error.
+        # One of very few cases where activation may encounter an error.
+        # N.B. doesn't really test rules-activation, but maybe worth doing.
+        # (no rules trigger)
+        with self.assertRaisesRegex(ValueError, "No ellipsoid"):
+            self.run_testcase(mapping_missingradius=True)
+
+    def test_bad_gridmapping_nameproperty(self):
+        # Fix the 'grid' var so it does not register as a grid-mapping.
+        # Rules Triggered:
+        #     001 : fc_default
+        #     002 : fc_provides_grid_mapping --FAILED(no grid-mapping attr)
+        #     003 : fc_provides_coordinate_(latitude)
+        #     004 : fc_provides_coordinate_(longitude)
+        #     005 : fc_build_coordinate_(latitude)(no-cs)
+        #     006 : fc_build_coordinate_(longitude)(no-cs)
+        # Notes:
+        #     grid-mapping: NONE
+        #     dim-coords: lat+lon
+        #     coords-build: latlon coords NO coord-system
+        result = self.run_testcase(gridmapvar_mappropertyname="mappy")
+        self.check_result(result, cube_no_cs=True)
+
+    def test_latlon_bad_gridmapping_varname(self):
+        # rename the grid-mapping variable so it is effectively 'missing'
+        # (I.E. the var named in "data-variable:grid_mapping" does not exist).
+        # Rules Triggered:
+        #     001 : fc_default
+        #     002 : fc_provides_coordinate_(latitude)
+        #     003 : fc_provides_coordinate_(longitude)
+        #     004 : fc_build_coordinate_(latitude)(no-cs)
+        #     005 : fc_build_coordinate_(longitude)(no-cs)
+        # Notes:
+        #     no coord-system
+        #     all the same as test_bad_gridmapping_nameproperty
+        warning = "Missing.*grid mapping variable 'grid'"
+        result = self.run_testcase(warning=warning, gridmapvar_name="grid_2")
+        self.check_result(result, cube_no_cs=True)
+
+    def test_latlon_bad_latlon_unit(self):
+        # Check with bad latitude units : 'degrees' in place of 'degrees_north'.
+        #
+        # Rules Triggered:
+        #     001 : fc_default
+        #     002 : fc_provides_grid_mapping_(latitude_longitude)
+        #     003 : fc_default_coordinate_(provide-phase)
+        #     004 : fc_provides_coordinate_(longitude)
+        #     005 : fc_build_coordinate_(miscellaneous)
+        #     006 : fc_build_coordinate_(longitude)
+        # Notes:
```

Comment:
Is there some logic to the way these notes are formatted? There seems to be some inconsistency with the indenting. Perhaps this is a bit nitpicky, but I think with the use of "=" as what I read as some kind of shorthand, keeping the formatting more consistent may improve readability. Also, what exactly does "=" mean here?

Reply:
Ok I will review all of these and try to make them consistent + clearer.